include/dsn/dist/replication/replication.codes.h

```diff
@@ -37,12 +37,15 @@ DEFINE_THREAD_POOL_CODE(THREAD_POOL_COMPACT)
 DEFINE_THREAD_POOL_CODE(THREAD_POOL_INGESTION)
 DEFINE_THREAD_POOL_CODE(THREAD_POOL_SLOG)
 DEFINE_THREAD_POOL_CODE(THREAD_POOL_PLOG)
+DEFINE_THREAD_POOL_CODE(THREAD_SLOW_QUERY)
```

Comment:
why name `THREAD_SLOW_QUERY`

Reply:
And the name should like `THREAD_POOL_...`

Reply:
THREAD_POOL_SCAN ?